todo-backend/README.adoc

```diff
@@ -297,16 +311,87 @@ $ curl -X POST -H "Content-Type: application/json"  -d '{"title": "This is my fi
 $ curl http://localhost:8080
 [{"completed":false,"id":1,"order":0,"title":"This is my first todo item!","url":"https://localhost:8080/1"}
 ----
+
+== Run the Backend Locally
+
+=== Package the Backend
+
+The backend is packaged and deployed on a provisioned server:
+
+[source,options="nowrap"]
+----
+$ mvn clean package -Pprovisioned-server
+----
+
+=== Run a Local PostgreSQL Database
+
+Before running the backend locally, we need to have a local PostgreSQL database that we can connect to.
+We use the `postgresql` docker image to create one:
+
+[source,options="nowrap"]
+----
+$ docker run --name todos-db \
+  -e POSTGRES_USER=todos \
+  -e POSTGRES_PASSWORD=mysecretpassword \
+  -p 5432:5432 \
+  postgres
+----
+
+This will create a database named `todos-db` that we can connect to on `localhost:5432` with the credentials `todos / mysecretpassword`.
+
+=== Run the Application
+
+With the PostgreSQL database running, we can start the backend by passing the required environment variables to connect to the database:
+
+[source,options="nowrap"]
+----
+$ JBOSS_HOME=./target/server \
+  POSTGRESQL_DATABASE=todos \
+  POSTGRESQL_SERVICE_HOST=localhost \
+  POSTGRESQL_SERVICE_PORT=5432 \
+  POSTGRESQL_USER=todos \
+  POSTGRESQL_PASSWORD=mysecretpassword \
+  POSTGRESQL_DATASOURCE=ToDos \
+  ./target/server/bin/standalone.sh
+...
+14:41:58,111 INFO  [org.jboss.as.server] (Controller Boot Thread) WFLYSRV0010: Deployed "todo-backend.war" (runtime-name : "todo-backend.war")
+...
+----
+
+The backend is running, and we can use the HTTP API to manage a list of todos:
+
+[source,options="nowrap"]
+----
+# get a list of todos
+$ curl http://localhost:8080/todo-backend
+[]
+
+# create a todo with the title "This is my first todo item!"
+$ curl -X POST -H "Content-Type: application/json"  -d '{"title": "This is my first todo item!"}' http://localhost:8080/todo-backend
+{"completed":false,"id":1,"order":0,"title":"This is my first todo item!","url":"https://localhost:8080/todo-backend/1"}%
+
+# get a list of todos with the one that was just created
+$ curl http://localhost:8080/todo-backend
+[{"completed":false,"id":1,"order":0,"title":"This is my first todo item!","url":"https://localhost:8080/todo-backend/1"}
+----
 endif::[]
 
+// Run the Arquillian Tests
+include::../shared-doc/run-arquillian-functional-tests-remote.adoc[leveloffset=+1]
```

Comment:
The instructions should clearly state that an user must start a server (as a Bootable Jar or as a provisioned one) in order to run the functional tests successfully.